src/main/g8/build.sbt

```diff
@@ -1,10 +1,6 @@
 name := "$name;format="lower,hyphen"$"
 version := "0.0.1-SNAPSHOT"
 
-scalaVersion := "2.13.10"
+scalaVersion := "3.3.6"
```

Comment:
I decided to stick to LTS, but we may use the current **Scala Next** as well.
I personally, would wait for `3.9` and then update again.